### ISSUE TITLE
vg call  report an error： vg: src/snarl_caller.cpp:501

### ISSUE THREAD

**Issue by WWilen** [vgteam/vg#3489]
**1. What were you trying to do?**
Try to genotype SVs from the short-reads  

**2. What did you want to happen?**
get a output in VCF format

**3. What actually happened?**
vg: src/snarl_caller.cpp:501: virtual std::pair<std::vector<int>, std::unique_ptr<vg::SnarlCaller::CallInfo> > vg::PoissonSupportSnarlCaller::genotype(const vg::Snarl&, const std::vector<vg::SnarlTraversal>&, int, int, const string&, std::pair<long unsigned int, long unsigned int>): Assertion `ploidy == 2 || ploidy == 1' failed.
ERROR: Signal 6 occurred. VG has crashed. Visit https://github.com/vgteam/vg/issues/new/choose to report a bug.
Stack trace path: /tmp/vg_crash_GyNPsS/stacktrace.txt
Please include the stack trace file in your bug report!

**4. If you got a line like `Stack trace path: /somewhere/on/your/computer/stacktrace.txt`, please copy-paste the contents of that file here:**

```
/tmp/vg_crash_GyNPsS/stacktrace.txt: No such file or directory
```

**5. What data and command can the vg dev team use to make the problem happen?**

**6. What does running `vg version` say?**

```
vg version v1.35.0 "Ghizzano"
Compiled with g++ (Ubuntu 10.3.0-1ubuntu1) 10.3.0 on Linux
Linked against libstd++ 20210408
Built by xian@sept
e
```

**Comment by glennhickey** [comment]
This should be fixed in #3490

**Comment by WWilen** [comment]
> This should be fixed in #3490
Thank you for your prompt reply, you means that i should update the VG version and use the vg call  -A parameter?
I am a novice

**Comment by glennhickey** [comment]
No, just running your command as is should work in the next release.  If you want, you can try a patched version here http://public.gi.ucsc.edu/~hickey/vg

**Comment by WWilen** [comment]
ok, thx

**Comment by faithokamoto** [comment]
Seems this was answered at the time.
